lua/entities/gmod_wire_expression2/core/custom/cfc_geometry_trig.lua

```diff
@@ -0,0 +1,38 @@
+E2Lib.RegisterExtension( "cfc_e2_lib", true )
+
+__e2setcost(4) --arbitrary
+
+local deg2rad = pi / 180
+local rad2deg = 180 / pi
+
+e2function vector2 polar2DtoGrid2DDeg(number distance, number ang)
+    return {
+            distance * cos(ang * deg2rad),
```

Comment:
you never defined cos. dont you need to define a local variable cos `local cos = math.cos` or just use math.cos?

same with sin

Reply:
Yes... i completely missed that.